### PR TITLE
Add buttons and align content

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update ProductCardBlock with buttons

Screenshot 2023-01-24 at 16 36 38
Screenshot 2023-01-24 at 16 36 56

Next step:
- Make "Calculate price" button open PriceCalculator on ProductPage

## Justify why they are needed
Updated design
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1433** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1433" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1433?utm_source=stack-comment).

File: apps/store/src/components/ProductCard/ProductCard.tsx [hunk 1]
```diff
@@ -14,23 +15,34 @@ export type ProductCardProps = {
...
-      <Space y={0.125}>
+      <ContentWrapper>
         <Title>{title}</Title>
         <Subtitle>{subtitle}</Subtitle>
-      </Space>
+        <CallToAction>
+          <Button href={link} size="medium" variant="secondary">
+            {t('READ_MORE')}
+          </Button>
+          <Button href={link} size="medium" variant="primary-alt">
```
Comment (alebedev) [inline apps/store/src/components/ProductCard/ProductCard.tsx:41]: How do you intend to open price calculator here?  Right now it simply opens product page, same as first button

When we have this working, I want to update price calculator tracking to ensure we're handling this scenario too

**Review by alebedev — CHANGES_REQUESTED** [review]
Looks good visually, see questions about opening price calculator

File: apps/store/src/components/ProductCard/ProductCard.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/ProductCard/ProductCard.tsx:41]: Good question, me and @robinandeer talked about adding a query param to the url and check for it in PriceCalculator, and toggle the modal. Or do you have another suggestion? :) @alebedev

Comment (gustaveen) [reply apps/store/src/components/ProductCard/ProductCard.tsx:41]: And I marked it as next step so will do it in a separate PR, but I can remove the button from the PR though 👍

**Review by alebedev — APPROVED** [review]

**Pull request merged**